src/main/frontend/components/diff.cljs

```diff
@@ -197,9 +199,9 @@
             path))
         [:div
          (ui/textarea
-          {:placeholder "Commit message (optional)"
-           :on-change (fn [e]
-                        (reset! commit-message (util/evalue e)))})
+           {:placeholder "Commit message (optional)"
```

Comment:
Currently the majority of code uses [single space indentation](https://github.com/bbatsov/clojure-style-guide#one-space-indent).

Reply:
exactly, I'll check my IDE about indentation.